From a5465ef6464f100ba118b29b4af10c5ff8debb95 Mon Sep 17 00:00:00 2001
From: "haolong.zhang" <haolong.zhang@ck-telecom.com>
Date: Mon, 6 Jul 2015 11:23:45 +0800
Subject: [PATCH]     [BugFix] min cpufreq table to 0.806G & remove Powerkey
 hang WD detect
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

    [id]:90831

    [TYPE]
    	BugFix

    [Products]
    	hicam_baseline
    [Requirement]
    	N/A
    [Solution Description]
    	N/A
    [Other info]
 要提交的变更：
	修改:         mediatek/kernel/drivers/wdk/wd_common_drv.c
	修改:         mediatek/platform/mt6595/kernel/core/mt_cpufreq.c

Change-Id: I097f80c0e2a07fc93cdc2ab87d30acc58bd5e507
---
 mediatek/kernel/drivers/wdk/wd_common_drv.c       | 20 ++++++++++----------
 mediatek/platform/mt6595/kernel/core/mt_cpufreq.c |  8 ++++----
 2 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/mediatek/kernel/drivers/wdk/wd_common_drv.c b/mediatek/kernel/drivers/wdk/wd_common_drv.c
index ef93159..253cdd8 100755
--- a/mediatek/kernel/drivers/wdk/wd_common_drv.c
+++ b/mediatek/kernel/drivers/wdk/wd_common_drv.c
@@ -384,16 +384,16 @@ static int kwdt_thread(void *arg)
 		
 		msleep((g_kinterval) * 1000);
 
-		#ifdef CONFIG_MTK_AEE_POWERKEY_HANG_DETECT
-		if( (cpu==0)&&(wk_tsk[cpu]->pid == current->pid))//only effect at cpu0
-		{
-			if(aee_kernel_wdt_kick_api(g_kinterval)==WDT_PWK_HANG_FORCE_HWT)
-			{
-				printk_sched("power key trigger HWT\n");
-				cpus_kick_bit=0xFFFF; //Try to force to HWT
-			}
-		}
-		#endif
+//		#ifdef CONFIG_MTK_AEE_POWERKEY_HANG_DETECT
+//		if( (cpu==0)&&(wk_tsk[cpu]->pid == current->pid))//only effect at cpu0
+//		{
+//			if(aee_kernel_wdt_kick_api(g_kinterval)==WDT_PWK_HANG_FORCE_HWT)
+//			{
+//				printk_sched("power key trigger HWT\n");
+//				cpus_kick_bit=0xFFFF; //Try to force to HWT
+//			}
+//		}
+//		#endif
 	}
 	printk("[WDK] WDT kicker thread stop, cpu:%d, pid:%d\n", cpu, current->pid);
 	return 0;
diff --git a/mediatek/platform/mt6595/kernel/core/mt_cpufreq.c b/mediatek/platform/mt6595/kernel/core/mt_cpufreq.c
index 1d9c5d3..f1a9898 100755
--- a/mediatek/platform/mt6595/kernel/core/mt_cpufreq.c
+++ b/mediatek/platform/mt6595/kernel/core/mt_cpufreq.c
@@ -110,7 +110,7 @@ extern void hp_limited_cpu_num(int num); // TODO: ask Marc to provide the head f
 #define DVFS_BIG_F4 (1144000) /* KHz */
 #define DVFS_BIG_F5 (1001000) /* KHz */
 #define DVFS_BIG_F6 (806000)  /* KHz */
-#define DVFS_BIG_F7 (403000)  /* KHz */
+#define DVFS_BIG_F7 (806000)  /* KHz */
 
 #if defined(SLT_VMAX)
 #define DVFS_BIG_V0 (1150)    /* mV */
@@ -123,7 +123,7 @@ extern void hp_limited_cpu_num(int num); // TODO: ask Marc to provide the head f
 #define DVFS_BIG_V4 (1000)    /* mV */
 #define DVFS_BIG_V5 (963)     /* mV */
 #define DVFS_BIG_V6 (914)     /* mV */
-#define DVFS_BIG_V7 (814)     /* mV */
+#define DVFS_BIG_V7 (914)     /* mV */
 
 #define DVFS_LITTLE_F0 (1690000) /* KHz */
 #define DVFS_LITTLE_F1 (1495000) /* KHz */
@@ -132,7 +132,7 @@ extern void hp_limited_cpu_num(int num); // TODO: ask Marc to provide the head f
 #define DVFS_LITTLE_F4 (1144000) /* KHz */
 #define DVFS_LITTLE_F5 (1001000) /* KHz */
 #define DVFS_LITTLE_F6 (806000)  /* KHz */
-#define DVFS_LITTLE_F7 (403000)  /* KHz */
+#define DVFS_LITTLE_F7 (806000)  /* KHz */
 
 #if defined(SLT_VMAX)
 #define DVFS_LITTLE_V0 (1150)    /* mV */
@@ -145,7 +145,7 @@ extern void hp_limited_cpu_num(int num); // TODO: ask Marc to provide the head f
 #define DVFS_LITTLE_V4 (1000)    /* mV */
 #define DVFS_LITTLE_V5 (963)     /* mV */
 #define DVFS_LITTLE_V6 (914)     /* mV */
-#define DVFS_LITTLE_V7 (814)     /* mV */
+#define DVFS_LITTLE_V7 (914)     /* mV */
 
 #define PWR_THRO_MODE_LBAT_1365MHZ	BIT(0)
 #define PWR_THRO_MODE_BAT_OC_806MHZ	BIT(1)
-- 
1.9.1

